coral/src/app/features/browse-topics/BrowseTopics.tsx

```diff
@@ -38,12 +38,12 @@ function BrowseTopics() {
   return (
     <>
       <Flexbox direction={"column"} gap={"l1"}>
```

Comment:
Maybe having the max width on this container is marginally better? 
```
        <Flexbox direction={"row"} colGap={"l1"} maxWidth={"6xl"}>
```

Reply:
That container is also wrapping the list, so that would be affected then, too.